src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -70,6 +73,19 @@ class WorkspaceService(protected val argUserToken: WithAccessToken, val rawlsDAO
 
   }
 
+  def getStorageCostEstimate(workspaceNamespace: String, workspaceName: String) = {
+    Future.sequence(Seq(
+      rawlsDAO.getBucketUsage(workspaceNamespace, workspaceName),
+      googleServicesDAO.fetchPriceList
+    )) map {
+      case Seq(usage: RawlsBucketUsageResponse, priceList: GooglePriceList) =>
+        val rate = priceList.prices.cpBigstoreStorage.us
+        val estimate: BigDecimal = BigDecimal(usage.usageInBytes) / 1000000000 * rate
+        RequestComplete(WorkspaceStorageCostEstimate(f"$$$estimate%.2f"))
```

Comment:
Choosing to fully format the dollar value server-side to avoid having to do any rounding/formatting client-side. Also, the price list currently only gives us US dollars, so there are no other formatting options.